src/backend/app/db/database.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+import os
+
+DB_URL = os.getenv("DATABASE_URL")
```

Comment:
here instead of getting the URL get the DEBUG and associated with it create a corresponding URL for the DB